### DISCUSSION TITLE
How can i disable just a part of a utility

### DISCUSSION THREAD

**Discussion by yassinebridi** [tailwindlabs/tailwindcss#3076] [Help] [answered]
I want to change just a part of the [inset](https://tailwindcss.com/docs/top-right-bottom-left) utilities, particularly the left and right utilities.
So if i disable the the inset utility like this:
```json
 corePlugins: {
   inset: false
 }
```
it will disable all the utilities including `top`, `right`...

**Comment by simonswiss** [comment, marked as answer]
Hey! 

I _don't think_ this is possible - you would need to disabled the entire `inset` core plugin and re-implement it the way you want it to work.

**Reply by yassinebridi** [reply]
That's my only option, thanks @simonswiss.
